ftl/qt/qt-misc.ftl

```diff
@@ -1,6 +1,6 @@
 qt-misc-addon-will-be-installed-when-a = Add-on will be installed when a profile is opened.
 qt-misc-addons = Add-ons
-qt-misc-all-cards-notes-and-media-for = All cards, notes, and media for this profile will be deleted. Are you sure?
+qt-misc-all-cards-notes-and-media-for = All cards, notes, and media for the profile "{ $name }" will be deleted. Are you sure?
```

Comment:
Please leave the old translation and add a new one, so that translators get notified that an update is required.

```suggestion
qt-misc-all-cards-notes-and-media-for2 = All cards, notes, and media for the profile "{ $name }" will be deleted. Are you sure?
```

Reply:
You don't have to worry about other languages. They are managed in a [separate repo](https://github.com/ankitects/anki-desktop-ftl) that's synced here regularly.